File: documentation-examples-scala/src/scala/dev/typr/foundations/docs/mariadb/IntegerTypesUnsigned.scala
package dev.typr.foundations.docs.mariadb
import dev.typr.scalafoundations.*
import dev.typr.scalafoundations.data.*



@SuppressWarnings(Array("unused"))
object IntegerTypesUnsigned:
  //start
  val unsignedTiny: MariaType[Uint1] = MariaTypes.tinyintUnsigned
  val unsignedInt: MariaType[Uint4] = MariaTypes.intUnsigned
  val unsignedBig: MariaType[Uint8] = MariaTypes.bigintUnsigned
  //stop

File: documentation-examples-java/src/java/dev/typr/foundations/docs/mariadb/IntegerTypesUnsigned.java
package dev.typr.foundations.docs.mariadb;

import dev.typr.foundations.MariaType;
import dev.typr.foundations.MariaTypes;
import dev.typr.foundations.data.Uint1;
import dev.typr.foundations.data.Uint4;
import dev.typr.foundations.data.Uint8;

@SuppressWarnings("unused")
public class IntegerTypesUnsigned {
  // start
  MariaType<Uint1> unsignedTiny = MariaTypes.tinyintUnsigned;
  MariaType<Uint4> unsignedInt = MariaTypes.intUnsigned;
  MariaType<Uint8> unsignedBig = MariaTypes.bigintUnsigned;
  // stop
}
